### ISSUE TITLE
[BUG] Findfbgl.cmake not found

### ISSUE THREAD

**Issue by dario-loi** [lvntky/fbgl#2]
When configuring the `example` directory's CMakeLists.txt, the Findfbgl.cmake module is not present:

```
cmake -G Ninja -S . -B build -DCMAKE_EXPORT_COMPILE_COMMANDS=ON
-- The C compiler identification is GNU 14.2.1
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/lib/ccache/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
CMake Error at CMakeLists.txt:5 (include):
  include could not find requested file:

    ../cmake/folders.cmake


CMake Error at CMakeLists.txt:8 (find_package):
  By not providing "Findfbgl.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "fbgl", but
  CMake did not find one.

  Could not find a package configuration file provided by "fbgl" with any of
  the following names:

    fbglConfig.cmake
    fbgl-config.cmake

  Add the installation prefix of "fbgl" to CMAKE_PREFIX_PATH or set
  "fbgl_DIR" to a directory containing one of the above files.  If "fbgl"
  provides a separate development package or SDK, be sure it has been
  installed.


-- Configuring incomplete, errors occurred!
```

This makes it impossible to proceed with building the example directory.

**Comment by lvntky** [comment]
Yes unfortunately i forgot to remove previous CMakeLists maybe we can remove the entire thing and write simple Makefile or sh to make executables of test programs. I can assigned it to you if you are interested

**Comment by dario-loi** [comment]
New CMakeLists merged in #8, closing.
